docs/source/tracking.rst

```diff
@@ -250,17 +250,19 @@ You run an MLflow tracking server using ``mlflow server``.  An example configura
 Storage
 -------
 
-An MLflow tracking server has two properties related to how data is stored: file store and artifact store.
+An MLflow tracking server has two properties related to how data is stored: a backend store and
+an artifact store.
 
 The *backend store* (exposed as ``--backend-store-uri``) is where the *server* stores run and
```

Comment:
In general, I find this section hard to read because it interleaves the option definition with definitions for the different types of stores. Can we split this up? We could have one paragraph defining the different stores and a separate paragraph about how to specify the type of store to use with the `--backend-store-uri` flag, including the flag's default value.

Reply:
Fixed ... this is how it now reads

![image](https://user-images.githubusercontent.com/6900999/54240100-5cae5900-44da-11e9-9c66-165379670943.png)